### DISCUSSION TITLE
"Next" button to advance to new game

### DISCUSSION THREAD

**Discussion by Designsbyvas** [octokatherine/word-master#41] [Ideas]
It would be nice if, after solving a puzzle, we could take a screenshot of the game or show it to someone in the room before moving to the next game. Adding an "advance to next game" action instead of automatically moving to the next screen would solve for this.

**Comment by octokatherine** [comment]
Yep! For now, you can click outside of the modal to view the game results and screenshot, then refresh the page to play again (you won't lose your streak)

I'll work on making a more clear way to do this in the interface

**Reply by Karyudo** [reply]
Thanks for the hint! To be honest, I now see that Word Master works *exactly* like Wordle does, except for one thing: Wordle doesn't hide 100% of the completed game when it puts up its modal/congratulation screen. Somehow that was enough for me to figure out to click outside the modal in Wordle, while I neglected to do the same thing in Word Master.

**Reply by Claredue** [reply]
Thank you for that hint. Just love the game but want to be able to look back at the steps I have taken to get to the answer.
